lib/generators/active_admin/assets/templates/tailwind.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
     './app/admin/**/*.{arb,erb,html,rb}',
     './app/views/active_admin/**/*.{arb,erb,html,rb}',
     './app/views/admin/**/*.{arb,erb,html,rb}',
+    './app/views/layouts/active_admin*.{arb,erb,html,rb}',
```

Comment:
The other paths can include more file extensions but the layouts will not and will only be `.html.erb` files.

```suggestion
    './app/views/layouts/active_admin*.{erb,html}',
```